content/staff-wellbeing/establish-a-wellbeing-committee.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Establish a wellbeing committee
+layout: /markdown_resource.html.erb
+colour: blue
```

Comment:
I went with an additional front matter value rather than an whole new layout because it's only the colour that changes.